contrib/release-tools/release.sh

```diff
@@ -23,6 +24,12 @@ function common_finalize {
 
 trap common_finalize EXIT
 
+# Don't release if the git repo has uncommitted changes
+git diff --quiet || error_out code=127 message="There are uncommitted changes. Refusing to release."
+
+git remote get-url upstream |grep "fedora-modularity/libmodulemd.git\$" -q || error_out code=127 message="There must be a git remote named 'upstream' available. The user running this script must have privilege to push commits to the 'upstream' remote"
```

Comment:
```suggestion
[[ $(git remote get-url upstream) =~ fedora-modularity/libmodulemd.git$ ]] || error_out code=127 message="There must be a git remote named 'upstream' available. The user running this script must have privilege to push commits to the 'upstream' remote"
```
As lone as you're using `bash`, you can do the regex check right in the shell. (This is optional, of course.)